### PR TITLE
[PM-4187] update `bwi-font` to v1.0.8

### PR DESCRIPTION
**Pull request by coroiu**
## Type of change

```
- [ ] Bug fix
- [x] New feature development
- [ ] Tech debt (refactoring, code cleanup, dependency upgrades, etc)
- [ ] Build/deploy pipeline (DevOps)
- [ ] Other
```

## Objective
Update `bwi-icon` to v1.0.8 to add support for `bwi-passkey` and `bwi-lock-encrypted`

## Before you submit

- Please add **unit tests** where it makes sense to do so (encouraged but not required)
- If this change requires a **documentation update** - notify the documentation team
- If this change has particular **deployment requirements** - notify the DevOps team
- Ensure that all UI additions follow [WCAG AA requirements](https://contributing.bitwarden.com/contributing/accessibility/)

### CODE REVIEW

File: libs/components/src/stories/icons.mdx [hunk 1]
```diff
@@ -44,49 +44,50 @@ or an options menu icon.
...
+| <i class="bwi bwi-envelope"></i>       | bwi-envelope       | action related to emailing a user            |
+| <i class="bwi bwi-external-link"></i>  | bwi-external-link  | open in new window or popout                 |
+| <i class="bwi bwi-eye"></i>            | bwi-eye            | show icon for password fields                |
+| <i class="bwi bwi-eye-slash"></i>      | bwi-eye-slash      | hide icon for password fields                |
+| <i class="bwi bwi-files"></i>          | bwi-files          | clone action / duplicate an item             |
+| <i class="bwi bwi-generate"></i>       | bwi-generate       | generate action in edit item forms           |
+| <i class="bwi bwi-generate-f"></i>     | bwi-generate-f     | generate feature or action                   |
+| <i class="bwi bwi-lock"></i>           | bwi-lock           | lock vault action                            |
+| <i class="bwi bwi-lock-f"></i>         | bwi-lock-f         | -                                            |
+| <i class="bwi bwi-lock-encrypted"></i> | bwi-lock-encrypted | -                                            |
```
Comment (coroiu) [inline libs/components/src/stories/icons.mdx:66]: `bwi-lock-encrypted` was just 1 space too big to fit, so the whole table got resized :shrug:

File: libs/components/src/stories/icons.mdx [hunk 2]
```diff
@@ -176,6 +177,7 @@ or an options menu icon.
 | <i class="bwi bwi-user-f"></i>            | bwi-user-f            | -                                              |
 | <i class="bwi bwi-wireless"></i>          | bwi-wireless          | -                                              |
 | <i class="bwi bwi-wrench"></i>            | bwi-wrench            | tools or additional configuration options      |
+| <i class="bwi bwi-passkey"></i>           | bwi-passkey           | passkey                                        |
```
Comment (Hinton) [inline libs/components/src/stories/icons.mdx:180]: Sort alphabetical?

Comment (coroiu) [reply libs/components/src/stories/icons.mdx]: Ah, totally missed the sorting 🤦 Fixed!

**Review by Hinton — APPROVED** [review]

**Comment by coroiu** [comment]
The changes had apparently already been merged as part of https://github.com/bitwarden/clients/pull/5616 so this PR now only contains the documentation updates that were missed

**Pull request merged**